### PR TITLE
Feature/protolint

### PR DESCRIPTION
**Pull request by davidhart82**
Add pareser for protolint

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Provides a parser and customized messages for ProtoLint.
+ *
+ * @author David Hart
+ */
+public class ProtoLint extends ReportScanningTool {
+
+	private static final long serialVersionUID = -5718503998068521571L;
+	
+	private static final String ID = "protolint";
```
[the remark this answers is not in the document]
Comment (davidhart82) [reply src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java]: Just clarifying, your second option is the same as the implemented one?  Or have I misunderstood?

File: src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        @NonNull
+        @Override
+        public String getDisplayName() {
+            return Messages.Warnings_ProtoLint_ParserName();
+        }
+
+        @Override
+        public boolean canScanConsoleLog() {
+            return false;
```
[the remark this answers is not in the document]
Comment (davidhart82) [reply src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java]: Done

File: src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        }
+
+        @Override
+        public String getUrl() {
+            return "https://github.com/yoheimuta/protolint";
+        }
+
+        @Override
+        public StaticAnalysisLabelProvider getLabelProvider() {
```
[the remark this answers is not in the document]
Comment (davidhart82) [reply src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java]: Done

File: src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+
+/**
+ * Provides a parser and customized messages for ProtoLint.
+ *
+ * @author David Hart
+ */
+public class ProtoLint extends ReportScanningTool {
+
+	private static final long serialVersionUID = -5718503998068521571L;
```
[the remark this answers is not in the document]
Comment (davidhart82) [reply src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java]: Sorry, yes, done.

**Comment by davidhart82** [comment]
I have no idea why this failed.

File: src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (uhafner) [reply src/main/java/io/jenkins/plugins/analysis/warnings/ProtoLint.java]: Ups, sorry. Seems that I did not wear my glasses. You are right, it already is a lower case `l`.

**Comment by uhafner** [comment]
Hmm, I also don't know what is wrong in GitHub Actions windows build, the command is `mvn -V clean verify --file pom.xml  -Djenkins.test.timeout=5000` but somehow the last option is not interpreted correctly...

**Pull request merged**